test/cluster/lwt/test_lwt_during_teblets_resize.py

```diff
@@ -0,0 +1,175 @@
+from test.cluster.lwt.lwt_common import (
+    BaseLWTTester,
+    wait_for_tablet_count,
+    DEFAULT_WORKERS,
+    DEFAULT_NUM_KEYS,
+)
+from test.cluster.util import new_test_keyspace
+from test.pylib.manager_client import ManagerClient
+from test.pylib.rest_client import inject_error_one_shot
+from test.pylib.tablets import get_tablet_count
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+# Test constants
+NUM_MIGRATIONS = 20
+WARMUP_LWT_CNT = 100
+POST_LWT_CNT = 100
+PHASE_WARMUP = 'warmup'
+PHASE_POST = 'post'
+PHASE_RESIZE = 'resize'
+
+
+@pytest.mark.asyncio
+@skip_mode("release", "error injections are not supported in release mode")
+@skip_mode("debug", "debug mode is too slow for this test")
+async def test_multi_column_lwt_during_split_merge(manager: ManagerClient):
+    """
+    Test scenario:
+      1. Start N servers with tablets enabled
+      2. Disable auto-balancing
+      3. Create keyspace/table
+      4. Insert rows, precompute pk->token
+      5. Start LWT workers
+      6. Run tablet resizing in parallel
+      7. Stop workers and verify consistency
+    """
+
+    cfg = {
+        "enable_tablets": True,
+        "tablet_load_stats_refresh_interval_in_seconds": 1,
+        "target-tablet-size-in-bytes": 1024 * 16,
+        "rf_rack_valid_keyspaces": False,
+    }
+
+    servers = await manager.servers_add(6, config=cfg)
+    target_server = servers[0]
+    # Disable balancer initially
+    logger.info(f"Disabling tablet balancing on random server: {target_server.ip_addr}")
+    await manager.api.disable_tablet_balancing(target_server.ip_addr)
+
+    rf_max = len(servers) - 1
+    rf = random.randint(2, rf_max)
+    logger.info("Using replication_factor=%d (servers=%d)", rf, len(servers))
+
+    async with new_test_keyspace(
+        manager,
+        f"WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': {rf}}} "
+        f"AND tablets = {{'initial': 1}}",
+    ) as ks:
+        stop_event_ = asyncio.Event()
+        table = "lwt_split_merge_table"
+        tester = BaseLWTTester(
+            manager,
+            ks,
+            table,
+            num_workers=DEFAULT_WORKERS,
+            num_keys=DEFAULT_NUM_KEYS,
+        )
+
+        await tester.create_schema()
+        await tester.initialize_rows()
+        await tester.start_workers(stop_event_)
+
+        try:
+            initial_cnt = await get_tablet_count(manager, servers[0], ks, table)
+            logger.info("Initial tablet_count=%d", initial_cnt)
+
+            # Phase 1: warmup LWT (100 applied CAS)
+            tester.set_phase(PHASE_WARMUP)
+            logger.info("LWT warmup: waiting for %d applied CAS", WARMUP_LWT_CNT)
+            await tester.wait_for_phase_ops(stop_event_, PHASE_WARMUP, WARMUP_LWT_CNT, timeout=180, poll=0.2)
+            logger.info("LWT warmup complete: %d ops", tester.get_phase_ops(PHASE_WARMUP))
+
+            # Force load stats refresh before enabling balancing
```

Comment:
nit: The comment doesn't add much, we can see what it describes. What we don't see, is why this needs to be done.